src/components/landing/Introduction.js

```diff
@@ -1,14 +1,46 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { theme } from '../../theme/docsTheme';
 import ComponentFactory from '../ComponentFactory';
 
-const Introduction = ({ nodeData: { children }, ...rest }) => (
-  <div className="introduction">
-    {children.map((child, i) => (
-      <ComponentFactory nodeData={child} key={i} {...rest} />
-    ))}
-  </div>
-);
+const Introduction = ({ nodeData: { children }, ...rest }) => {
+  const StyledIntroduction = styled('div')`
+    .button {
+      font-size: ${theme.fontSize.default};
+      height: unset;
+      margin-top: ${theme.size.medium};
+      margin-bottom: ${theme.size.default};
+      margin-right: ${theme.size.default};
```

Comment:
[nit] Could use `margin` shorthand
```suggestion
      margin: ${theme.size.medium} ${theme.size.default} ${theme.size.default} 0;
```